browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -723,7 +723,7 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
         Don't upgrade connections to HTTPS (disabled)
       </message>
       <message name="IDS_BRAVE_FORGET_FIRST_PARTY_STORAGE_SWITCH" desc="Text for First Party Storage cleanup shields setting.">
-        Forget me when I close this site
+        Forget me when I close every site, by default
```

Comment:
Defer to @rmcfadden3 here.

Reply:
Considering that this is a toggle "name" that's followed by the explanation `Clears cookies and other site data when you close a site`, I would just go for somethings like `Make sites forget me`, but again, defer to @rmcfadden3 here.

Reply:
Yeah, we don't want to imply it's Brave doing the forgetting. Also, I notice the other settings don't use pronouns like "I" or "me" in them. So I wonder if we should rather do:

How about `Clear session data when a site is closed`

Reply:
Updated and changed the screenshot.